src/oiiotool/expressions.cpp

```diff
@@ -317,6 +317,35 @@ Oiiotool::express_parse_atom(const string_view expr, string_view& s,
                 result = out.str();
                 if (result.size() && result.back() == '\n')
                     result.pop_back();
+
+            } else if (metadata == "IS_CONSTANT") {
+                std::vector<float> color((*img)(0, 0).nchannels());
+                if (ImageBufAlgo::isConstantColor((*img)(0, 0), color)) {
```

Comment:
I think this is not quite correct for the declaration of isConstantColor. I believe you want

```suggestion
                if (ImageBufAlgo::isConstantColor((*img)(0, 0), 0.0f, color)) {
```

The 2nd parameter is the threshold amount (0.0f would mean a "true constant", exact same value everywhere), and the 3rd parameter is the place to store the constant value it finds.